packages/Modal/src/Modal.js

```diff
@@ -71,8 +71,8 @@ const Modal = props => {
       onClose={onClose}
       onAfterOpen={onAfterOpen}
       onAfterClose={onAfterClose}
-      {...overlayProps}
       focusLockOptions={focusLockOptions}
+      {...overlayProps}
```

Comment:
I would think these props should be applied to override the focusLockOptions....

Reply:
why would be that? isn't we are having `<Modal.FocusLock  />` in charge of overriding the props?

Reply:
Ok, I see, I wasn't sure of the priority